src/PairInteractions.jl

```diff
@@ -130,78 +137,78 @@ function convert_quadratic(int::QuadraticInteraction{D}, cryst; tol=1e-6) where
 end
 
 function energy(spins::Array{Vec3}, heisenberg::HeisenbergCPU)
-    @unpack J, culled_bonds = heisenberg
+    @unpack effJ, culled_bonds = heisenberg
     E = 0.0
     latsize = size(spins)[2:end]
     for (i, bonds) in enumerate(culled_bonds)
         for bond in bonds
             @unpack j, n = bond
             for cell in CartesianIndices(latsize)
-                Sᵢ = spins[i, cell]
-                Sⱼ = spins[j, offset(cell, n, latsize)]
-                E += Sᵢ ⋅ Sⱼ
+                sᵢ = spins[i, cell]
+                sⱼ = spins[j, offset(cell, n, latsize)]
+                E += sᵢ ⋅ sⱼ
```

Comment:
Question: It seems sometimes we use lower case s and sometimes upper case S. Is it intended that these have subtly different meanings?

Reply:
Yeah, the convention I'm working with is that lower-case s are the _unit vectors_ we internally store. Upper case S in the code should be a scalar spin magnitude. In the math within comments/documentation, I try to use this convention:

```𝐒_i = S_i 𝐬_i```

So, upper-case bold S is the full vector, upper-case S is the scalar spin magnitude, lower-case is the unit vector.

Reply:
This sounds good, except we should not try to use bolded `𝐒` as an actual Julia variable identifier (which you don't, just commenting...)